src/platform/test/functional/apps/discover/tabs/_save_and_load.ts

```diff
@@ -73,6 +73,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     describe('multi-tab Discover sessions', () => {
       const sessionName = 'Multi-tab Discover session';
+      const nonAutoChartIntervalTitle = 'Hour';
+      let persistedTabChartIntervalValue: string;
```

Comment:
```suggestion
      const persistedTabChartIntervalTitle = 'Hour';
      const persistedTabChartIntervalValue = 'h';
```
Could we use a const instead of a mutable variable here? Seems like it would be better to use a hardcoded value for assertions since we know what it should be.